### PR TITLE
Added a 'no data' message when no values are present for social fields

### LINKED ISSUE #13667: [UX] Social tab appears blank when there is not info on contact details
When a contact does not have social information associated with them, a blank space is shown.
To improve the user experience, we could have a message of "Nothing found", or hide the tab.

How to get there:
1. Open Contacts
2. Create a new
3. Save & Close
4. Click on Details > Social

Image

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

### PR DESCRIPTION
**Pull request by shinde-rahul**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴🟢 
| New feature/enhancement? (use the a.x branch)      | 🔴🟢
| Deprecations?                          | 🔴🟢
| BC breaks? (use the c.x branch)        | 🔴🟢
| Automated tests included?              | 🔴🟢 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #13667 

## Description

---
### 📋 Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Contacts
3. Create a new
4. Save & Close
5. Click on Details > Social

### CODE REVIEW

File: app/bundles/LeadBundle/Translations/en_US/messages.ini [hunk 1]
```diff
@@ -132,6 +132,7 @@ mautic.lead.field.group.extra="Extra"
 mautic.lead.field.group.personal="Personal"
 mautic.lead.field.group.professional="Professional"
 mautic.lead.field.group.social="Social"
+mautic.lead.field.group.no_data="No datas"
```
Comment (escopecz) [inline app/bundles/LeadBundle/Translations/en_US/messages.ini]: I have ChatGPTed it 🙂 :

> The word "data" is already plural, so it doesn't need an "s" at the end. Additionally, "data" is often treated as an uncountable noun

```suggestion
mautic.lead.field.group.no_data="No data"
```

**Review by escopecz — CHANGES_REQUESTED** [review]
The code looks good. Just one wording change request.

File: app/bundles/LeadBundle/Translations/en_US/messages.ini (hunk 1 above)
Comment (shinde-rahul) [reply app/bundles/LeadBundle/Translations/en_US/messages.ini]: It was definitely a typo, or no command key when trying to save.

**Review by escopecz — APPROVED** [review]
I see no issue in the code. Thank you! 👍

**Comment by escopecz** [comment]
@rahuld-dev please fill in the steps to test. It will increase the chance to get this tested.

**Comment by rahuld-dev** [comment]
> @rahuld-dev please fill in the steps to test. It will increase the chance to get this tested.

@shinde-rahul 👆

**Comment by escopecz** [comment]
Apologies again, Rahuls!

**Review by andersonjeccel — APPROVED** [review]
Works as expected

image

**Pull request merged**